pact.openapi.yaml

```diff
@@ -347,13 +347,41 @@ components:
           items:
             properties:
               sig:
-                type: string
-                contentEncoding: base16
-                description: |
-                  Base16-encoded cryptograhic signature of `cmd` field data
-                  for corresponding signer in payload.
-                example:
-                  "8d452109cc0439234c093b5e204a7428bc0a54f22704402492e027aaa9375a34c910d8a468a12746d0d29e9353f4a3fbebe920d63bcc7963853995db015d060f"
+                anyOf:
+                  - type: string
+                    contentEncoding: base16
+                    description: |
+                      Base16-encoded cryptograhic signature of `cmd` field data
+                      for corresponding signer in payload.
+                    example:
+                      "8d452109cc0439234c093b5e204a7428bc0a54f22704402492e027aaa9375a34c910d8a468a12746d0d29e9353f4a3fbebe920d63bcc7963853995db015d060f"
+                  - $ref: '#/components/schemas/webauthn-sig-string'
+
+    webauthn-sig-string:
+      type: string
+      contentMediaType: application/json
```

Comment:
There is IIRC some tooling which doesn't work with `contentMediaType`, though this is still useful info. I recommend commenting this out but keeping it, like we do in `binaryHeader`.